tests/e2e/vmlistener_test.go

```diff
@@ -56,20 +58,12 @@ func (s *VMListenerTestParams) SetUpSuite(c *C) {
 	s.vm2 = s.config.DockerHosts[1]
 	s.vm1Name = s.config.DockerHostNames[0]
 	s.vm2Name = s.config.DockerHostNames[1]
+	s.vsanDSName = esxutil.GetDatastoreByType("vsan")
 }
 
 func (s *VMListenerTestParams) SetUpTest(c *C) {
 	s.volumeName = inputparams.GetUniqueVolumeName("vmlistener_test")
 	s.containerName = inputparams.GetUniqueContainerName("vmlistener_test")
-
-	out, err := dockercli.CreateVolume(s.vm1, s.volumeName)
-	c.Assert(err, IsNil, Commentf(out))
-
-	out, err = dockercli.AttachVolume(s.vm1, s.volumeName, s.containerName)
-	c.Assert(err, IsNil, Commentf(out))
-
-	status := verification.VerifyAttachedStatus(s.volumeName, s.vm1, s.esx)
-	c.Assert(status, Equals, true, Commentf("Volume %s is not attached", s.volumeName))
 }
 
 func (s *VMListenerTestParams) TearDownTest(c *C) {
```

Comment:
Ah, I forgot to remove this TearDownTest. I will take care of this in my follow-up PR.